src/Mobs/Wolf.cpp

```diff
@@ -18,7 +18,7 @@ cWolf::cWolf(void) :
 	m_IsBegging(false),
 	m_IsAngry(false),
 	m_OwnerName(""),
-	m_CollarColor(14)
+	m_CollarColor(E_META_DYE_ORANGE)
```

Comment:
Either way the other change (the 15-x one) is more important, and it'd be nice if someone would confirm why it was like that.